.github/modify_sitemap.js

```diff
@@ -45,13 +106,27 @@ fs.readFile(inputFile, "utf8", (err, data) => {
     if (locPattern.test(match) && !locDirectPattern.test(match)) {
       return ""; // Remove entire <url> block if <loc> contains an unwanted segment
     }
+    match.replace(linkPattern, (link) => {
+      console.log("Checking Link", link);
```

Comment:
why do we need a console in production, please remove.

Reply:
removed